### PR TITLE
[EVENTVWR] IDD_LOGPROPERTIES Add some non-MS-accelerators

### PR DESCRIPTION
**Pull request by JoachimHenze**
* Add some non-MS-accelerators into IDD_LOGPROPERTIES for some functionality where our eventvwr does outperform the MS version. We do have some edit-fields selectable, where MS decided to not have them selectable. Can be used for copying out text.
* Therefore we considered that it may be beneficial to have accelerators for them as well.
* I used &S &C &E &A (instead of Katayamas former &S &C &M &A because the M is already in use in that dlg) where that did fit, and tried to manage the other languages without introducing collisions
* I also added a textual disclaimer on how to deal with that for the translators and with the aim to protect that diff to MS
* also fix some inconsistencies in the length of the IDS_COPY lines for some languages e.g. cs-CZ, pt-PT, pl-PL
* fix some text cutoff for el-GR,pt-BR, pt-PT at the radio-buttons
* improve german Help-accel
* add some FIXMEs where some langs had accel collisions, let the native speakers with their native 2k3sp2 handle them later

JIRA issue: none

## TODO

- [x] added a comment to explain it 
- [x] merged it into all langs

### CODE REVIEW

File: base/applications/mscutils/eventvwr/lang/ja-JP.rc [hunk 1]
```diff
@@ -115,21 +115,24 @@ END
 IDD_LOGPROPERTIES_GENERAL DIALOGEX 0, 0, 252, 234
 STYLE DS_SHELLFONT | WS_POPUP | WS_CAPTION
 CAPTION "一般設定"
-FONT 8, "MS Shell Dlg"
+FONT 9, "MS UI Gothic", 400, 0, 0x1
```
Comment (JoachimHenze) [inline base/applications/mscutils/eventvwr/lang/ja-JP.rc:118]: Katayama, will this font-line-change give a good result? I just copy-pasted it from a neighbor dlg without testing it.

File: base/applications/mscutils/eventvwr/lang/ja-JP.rc [hunk 2]
```diff
@@ -115,21 +115,24 @@ END
...
     LTEXT "表示名(&D):", IDC_STATIC, 7, 9, 60, 8
     EDITTEXT IDC_DISPLAYNAME, 67, 7, 178, 12, ES_LEFT | ES_AUTOHSCROLL | WS_CHILD | WS_VISIBLE | WS_BORDER | WS_TABSTOP
     LTEXT "ログ名(&L):", IDC_STATIC, 7, 25, 60, 8
     EDITTEXT IDC_LOGNAME, 67, 23, 178, 12, ES_LEFT | ES_AUTOHSCROLL | ES_READONLY | WS_CHILD | WS_VISIBLE | WS_BORDER | WS_TABSTOP
     LTEXT "ログファイル(&F):", IDC_STATIC, 7, 41, 60, 8
     EDITTEXT IDC_LOGFILE, 67, 39, 178, 12, ES_LEFT | ES_AUTOHSCROLL | ES_READONLY | WS_CHILD | WS_VISIBLE | WS_BORDER | WS_TABSTOP
-    LTEXT "サイズ:", IDC_STATIC, 7, 57, 60, 8
+// The following 4 IDC_STATIC shall have an accelerator in ReactOS, although MS doesn't have accels for them.
+// Dear Translators: First fill out ALL accels that MS brings as well in this dialog for your language,
+// and only then as a final step use some remaining unused letters for those 4 controls!
```
Comment (JoachimHenze) [inline base/applications/mscutils/eventvwr/lang/ja-JP.rc:128]: Do you think this template is good? @katahiromz ? @HBelusca ?

File: base/applications/mscutils/eventvwr/lang/ja-JP.rc [hunk 3]
```diff
@@ -115,21 +115,24 @@ END
...
+    LTEXT "サイズ(&S):", IDC_STATIC, 7, 57, 60, 8
     EDITTEXT IDC_SIZE_LABEL, 67, 57, 178, 12, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
-    LTEXT "作成日時:", IDC_STATIC, 7, 69, 60, 8
+    LTEXT "作成日時(&C):", IDC_STATIC, 7, 69, 60, 8
     EDITTEXT IDC_CREATED_LABEL, 67, 69, 178, 12, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
-    LTEXT "変更日時:", IDC_STATIC, 7, 81, 60, 8
+    LTEXT "変更日時(&E):", IDC_STATIC, 7, 81, 60, 8
```
Comment (JoachimHenze) [inline base/applications/mscutils/eventvwr/lang/ja-JP.rc:133]: Notice that it is &E now, not &M. We do have to use a letter that MS did not use yet (for `LTEXT "最大ログサイズ(&M):", IDC_STATIC, 17, 122, 58, 8` ). I do think our new accels have to humbly fit in next to the existing MS-ones, not requesting any dominance over them. Ok?

**Comment by katahiromz** [comment]
## Tutorial of `RisohEditor` for testing Japanese dialogs in Windows

Step 0: Install `Droid Sans Fallback` font.
Step 1: Install `RisohEditor`.
Step 2: Open the target module.
tut1

Step 3: Set font substitute setting of `"MS UI Gothic"`.
tut2

Step 4: Open the dialog resource.
tut3

**Comment by JoachimHenze** [comment]
Thank you for your feedback @katahiromz . I will leave the font-change out then, because I don't have any clear understanding about how it is supposed to look, and leave that to the japanese professional(s) later.

File: base/applications/mscutils/eventvwr/lang/ja-JP.rc (hunk 1 above)
Comment (JoachimHenze) [reply base/applications/mscutils/eventvwr/lang/ja-JP.rc]: I reverted that part, because this is how understood Katayamas reply. But maybe he wanted to say the exact opposite, that my change was good. I cannot say. Resolved.

**Review by katahiromz — APPROVED** [review]
`ja-JP` is okay.

**Comment by HBelusca** [comment]
> Can be used for copying out text.

Yes this was indeed the intent.

File: base/applications/mscutils/eventvwr/lang/zh-TW.rc [hunk 4]
```diff
@@ -123,13 +123,16 @@ BEGIN
...
+    LTEXT "大小：(&S)", IDC_STATIC, 7, 57, 60, 8
     EDITTEXT IDC_SIZE_LABEL, 67, 57, 178, 12, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
-    LTEXT "建立日期：", IDC_STATIC, 7, 69, 60, 8
+    LTEXT "建立日期：(&C)", IDC_STATIC, 7, 69, 60, 8
     EDITTEXT IDC_CREATED_LABEL, 67, 69, 178, 12, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
-    LTEXT "修改日期：", IDC_STATIC, 7, 81, 60, 8
+    LTEXT "修改日期：(&E)", IDC_STATIC, 7, 81, 60, 8
     EDITTEXT IDC_MODIFIED_LABEL, 67, 81, 178, 12, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
-    LTEXT "存取日期：", IDC_STATIC, 7, 93, 60, 8
+    LTEXT "存取日期：(&A)", IDC_STATIC, 7, 93, 60, 8
```
Comment (HBelusca) [inline base/applications/mscutils/eventvwr/lang/zh-TW.rc:135]: As far as I can see elsewhere in this file and others, in chinese/japanese/korean, the accelerator would precede the double-points
(so here for example it would need to be:
```suggestion
    LTEXT "存取日期(&A)：", IDC_STATIC, 7, 93, 60, 8
```
)

File: base/applications/mscutils/eventvwr/lang/zh-HK.rc [hunk 5]
```diff
@@ -172,13 +175,13 @@ BEGIN
 /* Please note that explicit TAB characters '\t' are used to fix the alignment of the message in the MessageBox */
     IDS_USAGE "ReactOS Event Viewer\n\
 \n\
-EventVwr [電腦名稱] [/l:<事件記錄檔案>] [/?]\n\
+EventVwr [電腦名稱] [/L:<事件記錄檔案>] [/?]\n\
```
Comment (HBelusca) [inline base/applications/mscutils/eventvwr/lang/zh-HK.rc:178]: Please check tr-TR.rc it needs a similar adjustment.

File: base/applications/mscutils/eventvwr/lang/bg-BG.rc [hunk 6]
```diff
@@ -122,13 +122,16 @@ BEGIN
     EDITTEXT IDC_LOGNAME, 67, 23, 178, 12, ES_LEFT | ES_AUTOHSCROLL | ES_READONLY | WS_CHILD | WS_VISIBLE | WS_BORDER | WS_TABSTOP
     LTEXT "Log &file:", IDC_STATIC, 7, 41, 60, 8
     EDITTEXT IDC_LOGFILE, 67, 39, 178, 12, ES_LEFT | ES_AUTOHSCROLL | ES_READONLY | WS_CHILD | WS_VISIBLE | WS_BORDER | WS_TABSTOP
-    LTEXT "Size:", IDC_STATIC, 7, 57, 60, 8
+// The following 4 IDC_STATIC shall have an accelerator in ReactOS, although MS doesn't have accels for them.
+// Dear Translators: First fill out ALL accels that MS brings as well in this dialog for your language,
```
Comment (HBelusca) [inline base/applications/mscutils/eventvwr/lang/bg-BG.rc:126]: "Dear Translators:" sounds overpolite here. "Note for Translators:"...

File: base/applications/mscutils/eventvwr/lang/cs-CZ.rc [hunk 7]
```diff
@@ -11,24 +11,24 @@ IDM_EVENTVWR MENU
...
-        MENUITEM "&Otevřít...", IDM_OPEN_EVENTLOG
-        MENUITEM "Uložit &Protokol jako...", IDM_SAVE_EVENTLOG
+        MENUITEM "&Otevřít...", IDM_OPEN_EVENTLOG//FIXME: accel collision &O
+        MENUITEM "Uložit &Protokol jako...", IDM_SAVE_EVENTLOG//FIXME: accel collision &P
         MENUITEM "&Zavřít", IDM_CLOSE_EVENTLOG
         MENUITEM SEPARATOR
-        MENUITEM "&Odstranit události", IDM_CLEAR_EVENTS
-        MENUITEM "&Přejmenovat\tF2", IDM_RENAME_EVENTLOG
+        MENUITEM "&Odstranit události", IDM_CLEAR_EVENTS//FIXME: accel collision &O
+        MENUITEM "&Přejmenovat\tF2", IDM_RENAME_EVENTLOG//FIXME: accel collision &P
```
Comment (HBelusca) [inline base/applications/mscutils/eventvwr/lang/cs-CZ.rc:19]: Please add a space before the `//FIXME` (remark for all of these in the PR)

File: base/applications/mscutils/eventvwr/lang/cs-CZ.rc [hunk 8]
```diff
@@ -216,7 +219,8 @@ END
 
 STRINGTABLE
 BEGIN
-    IDS_COPY "Typ události:     %s\r\n\
+    IDS_COPY "\
+Typ:       %s\r\n\
```
Comment (HBelusca) [inline base/applications/mscutils/eventvwr/lang/cs-CZ.rc:223]: Why the change? You speak czech?

File: base/applications/mscutils/eventvwr/lang/de-DE.rc [hunk 9]
```diff
@@ -217,14 +220,15 @@ END
...
-Computer:       %s\r\n\
+    IDS_COPY "\
+Typ:         %s\r\n\
+Quelle:      %s\r\n\
+Kategorie:   %s\r\n\
+Ereignis-ID: %s\r\n\
+Datum:       %s\r\n\
+Zeit:        %s\r\n\
+Benutzer:    %s\r\n\
+Computer:    %s\r\n\
```
Comment (HBelusca) [inline base/applications/mscutils/eventvwr/lang/de-DE.rc:231]: The purpose of the "extra" spaces was to have the values (`%s\r\n`) aligned on tab-length boundaries, since this is copiable data (that gets stored in the registry) for the user..
So please revert this change (here and in the other files where this has been done)

**Review by HBelusca — CHANGES_REQUESTED** [review]

File: base/applications/mscutils/eventvwr/lang/zh-TW.rc (hunk 4 above)
Comment (JoachimHenze) [reply base/applications/mscutils/eventvwr/lang/zh-TW.rc]: Good catch, and easy to miss that on my host where the chinese ':' are only tofu-blocks. I actually am aware of the desired order. Resolved.
image

File: base/applications/mscutils/eventvwr/lang/zh-HK.rc (hunk 5 above)
Comment (JoachimHenze) [reply base/applications/mscutils/eventvwr/lang/zh-HK.rc:178]: Resolved.

File: base/applications/mscutils/eventvwr/lang/bg-BG.rc (hunk 6 above)
Comment (JoachimHenze) [reply base/applications/mscutils/eventvwr/lang/bg-BG.rc:126]: I acted on your suggestion, I used "Translation note". Resolved.

File: base/applications/mscutils/eventvwr/lang/cs-CZ.rc (hunk 7 above)
Comment (JoachimHenze) [reply base/applications/mscutils/eventvwr/lang/cs-CZ.rc:19]: I recently changed to that style without the space intentionally, because I found several translators not deleting the whitespace but just the part beginning with the slashes when acting on the FIXMEs. Reviewing that individually each time to prevent us sticking with that whitespace is not worth the hassle imho. Keep in mind that those comments are a **temporary thing** only, until they are fixed. Ultimately they will be gone, so I think this practice is acceptable.

File: base/applications/mscutils/eventvwr/lang/cs-CZ.rc (hunk 8 above)
Comment (JoachimHenze) [reply base/applications/mscutils/eventvwr/lang/cs-CZ.rc:223]: The události == event exceeded the available space here, and required the ':' of the first line to be unaligned to the other ones.

File: base/applications/mscutils/eventvwr/lang/de-DE.rc (hunk 9 above)
Comment (JoachimHenze) [reply base/applications/mscutils/eventvwr/lang/de-DE.rc:231]: Personally I don't see the value yet in aligning that to a multiple of 4 spaces, although I am/was aware of its usage for copying data out. To me aligning it at the rightmost ':' makes more sense, but I can restore the multiple-of-4-spaces if you consider that to be an absolute must.

Comment (JoachimHenze) [reply base/applications/mscutils/eventvwr/lang/de-DE.rc]: I followed your suggestion of restoring 4-space-indentation, but as you can see this still leads to changes in some of the languages, actually proving your claim that we would use 4-space-indentation here everywhere by intention to be wrong. I feel the result being very aesthetically unappealing now, and perceived the state of rightmost `: %s\r\n` that I used before vastly superior.

Comment (JoachimHenze) [reply base/applications/mscutils/eventvwr/lang/de-DE.rc]: Why does it have to be a multiple of 4? Can't we just use one space for all langs, and 2 spaces for french maybe as the outlier?

File: base/applications/mscutils/eventvwr/lang/cs-CZ.rc (hunk 7 above)
Comment (HBelusca) [reply base/applications/mscutils/eventvwr/lang/cs-CZ.rc:19]: ok, this is reasonable.

**Review by HBelusca — APPROVED** [review]

File: base/applications/mscutils/eventvwr/lang/de-DE.rc (hunk 9 above)
Comment (HBelusca) [reply base/applications/mscutils/eventvwr/lang/de-DE.rc]: > all of them changed now by following your suggestion == proving that they did not follow your suggestion beforehand

Because these have been translated _**after**_ the english and french translations have been made and nobody in those other languages respected the alignment thing.

Comment (JoachimHenze) [reply base/applications/mscutils/eventvwr/lang/de-DE.rc]: Yes, and I don't feel obliged to that artificial obligation neither, same as the other translators. Note, that I didn't transform anything away from the 4 spaces indentation in this PR! To respect your feelings. Everything I touched in that padding regard now with the final commit turned some which were `spaces module 4 != 0` before and became `spaces modulo 4 == 0` after. But like I said: I didn't turn any of them to that which would have required me to use the space key. I do hope that we can both live with that compromise for master. If you want to enforce the space-bloat everywhere, you will have to do that yourself.
